app/views/components/_dropdown.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  id ||= "image-meta-#{SecureRandom.hex(4)}"
```

Comment:
Looks like this is left over from a copy + paste (image-meta)? It doesn't look like we use this id.

Reply:
Oh.. seems like that. Will update now.